projects/demo/src/modules/tables/table/examples/5/index.ts

```diff
@@ -55,18 +55,15 @@ export class TuiTableExample5 {
 
     readonly columns = ['name', 'dob', 'age'];
 
-    readonly ageSorter: TuiComparator<User> = (a: User, b: User) => getAge(a) - getAge(b);
+    readonly ageSorter: TuiComparator<User> = (a: User, b: User) =>
+        this.getAge(a) - this.getAge(b);
```

Comment:
Same here. The idea of moving pure functions away from this is:
- We are sure they are pure
- Their names can be minified
- They do not pollute class API (not applicable here since they are still publicly exposed)